src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -117,7 +117,7 @@ public function printFormatPreserving(array $stmts, array $origStmts, array $ori
         // strip empty starting/ending php tags
         if (array_key_exists(0, $stmts) && $stmts[0] instanceof FileWithoutNamespace) {
             $content = Strings::replace($content, self::EMPTY_STARTING_TAG_REGEX, '');
-            $content = Strings::replace(\rtrim($content), self::EMPTY_ENDING_TAG_REGEX, '', ) ."\n";
+            $content = Strings::replace(\rtrim($content), self::EMPTY_ENDING_TAG_REGEX, '',) . "\n";
```

Comment:
oh interessting.. seems in my last PR I forgot a dangling `,`

```suggestion
            $content = Strings::replace(\rtrim($content), self::EMPTY_ENDING_TAG_REGEX, '') . "\n";
```